includes/admin/class-wcmp-admin.php

```diff
@@ -844,10 +888,16 @@ public static function renderTrackTraceLink(array $shipment, int $order_id): voi
 
     /**
      * @param array $shipment
+     * @param int   $order_id
      */
-    public static function renderStatus(array $shipment): void
+    public static function renderStatus(array $shipment, int $order_id): void
     {
         echo $shipment["status"] ?? "–";
+
+        if (strstr($shipment['status'], 'afgeleverd')) {
```

Comment:
Probeer van 'afgeleverd' een constante te maken

Reply:
moet afgeleverd niet in het Engels?

Reply:
De status die je terugkrijgt van $shipment['status'] is helaas in het nederlands.